### DISCUSSION TITLE
When will `streamdeckc` be included in the releases?

### DISCUSSION THREAD

**Discussion by dysbulic** [streamdeck-linux-gui/streamdeck-linux-gui#117] [Q&A]
I did a little program to do [dynamic recording and streaming buttons for OBS](https://github.com/dysbulic/stream-deck-obs-buttons#readme) using the recently added `streamdeckc`.

Currently the control program doesn't seem to be included in the distributions. I was wondering if there's a timeline for when it's going to be included.

I was also wondering why the decision was made to do the control via a Unix socket rather than D-Bus?

**Comment by coolapso** [comment]
> Currently the control program doesn't seem to be included in the distributions. I was wondering if there's a timeline for when it's going to be included.

not sure exactly what you mean with this, can you clarify?

> I was also wondering why the decision was made to do the control via a Unix socket rather than D-Bus?

About this, I have no particular insight, @asuka1975 took the initiative and offered is work and time to the cause, and it was added to the project, so maybe only it can answer your question with more detail.

**Comment by dysbulic** [comment]
> > Currently the control program doesn't seem to be included in the distributions. I was wondering if there's a timeline for when it's going to be included.
> 
> not sure exactly what you mean with this, can you clarify?

I think with the most recently released version, running `pip install streamdeck-linux-gui` will create the `streamdeckc` command which is what I needed for my software to function.

Thanks.

**Comment by asuka1975** [comment]
@dysbulic No special reason. I adopted it because I was personally familiar with it and preferred it.

**Reply by coolapso** [reply]
As far as I am concerned, its a perfectly valid reason :D
